client/src/components/comments/btn-group/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from '@/components/button';
+import { useTranslations } from 'next-intl';
+
+interface IEditorBtnGroup {
+  onSave: () => void;
+  onDisabled: () => boolean | undefined;
```

Comment:
Instead of passing the onDisabled function, which gets executed during rendering, it's better to pass a precomputed boolean value isDisabled.